mage_ai/data_preparation/models/pipeline.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, uuid, repo_path=None, config=None, repo_config=None, catalog=
         self.block_configs = []
         self.blocks_by_uuid = {}
         self.concurrency_config = dict()
+        self.created_at = datetime.datetime.utcnow()
```

Comment:
does it make sense to default this to current time?

Reply:
Initially, i thought it'd just be a placeholder since we read it from the metadata config file, but thanks for pointing it out since it might have unintended side effects whenever the Pipeline class is instantiated.